scss/_tables.scss

```diff
@@ -1,31 +1,42 @@
+// stylelint-disable selector-max-universal
 //
 // Basic Bootstrap table
 //
 
 .table {
+  --table-cell-padding: #{$table-cell-padding};
+  --table-border-width: 0;
+  --table-border-bottom-width: #{$table-border-width};
+  --table-bg: #{$table-bg};
+  --table-accent-bg: #{$table-accent-bg};
+  --table-hover-bg: #{$table-hover-bg};
+  --table-hover-color: #{$table-hover-color};
+  --table-active-bg: #{$table-active-bg};
+  --table-children-border-width: #{$table-children-border-width};
+
   width: 100%;
   margin-bottom: $spacer;
   color: $table-color;
   vertical-align: $table-cell-vertical-align;
-  background-color: $table-bg; // Reset for nesting within parents with `background-color`.
+  border-color: $table-border-color;
 
-  th,
-  td {
-    padding: $table-cell-padding;
-    border-bottom: $table-border-width solid $table-border-color;
+  // th & td
+  > * > * > * {
```

Comment:
Isn't this a bit too much?

Reply:
It's the most efficient way to target `td`s and `th`s of the current table. 

Alternative is:
```css
.table > thead > tr > td,
.table > thead > tr > th,
.table > tbody > tr > td,
.table > tbody > tr > th,
.table > tfoot > tr > td,
.table > tfoot > tr > th {
  ...
}
```
which is:
- more code
- less performant (negligible though)

Reply:
Perhaps this should be:

```scss
> * > * > td,
> * > * > th {
```

So that it doesn't affect content in the `<caption>` element (and possibly the `<colgroup>/<col>`)

Reply:
`colgroup/col` shouldn't be an issue. `> :not(caption) > * > *` could also hep us out here? (bummer this increases specificity)

Reply:
Yeah. `:not(caption)` might be simpler for  reducing the ruleset size